tests/test_board.py

```diff
@@ -0,0 +1,53 @@
+from context import src
+from src import board as Board
+from src import cell as Cell
+import unittest
```

Comment:
so you used unittest. That is fine, as it is the basic unit test suite that is included in the standard library.
But I wonder if we should use pytest instead, just because last time I checked it was meant to be more feature rich and very simple to use. AS a bonus, all unittest style tests should work out of the box in pytest. Worth checking.
One thing I think you can do with pytest and not with unittest is having "fixtures" i,e, parametrise a test and then in batch give it at lot of params.

Reply:
Adopted unittest